content/en/docs/v3.6/op-guide/configuration.md

```diff
@@ -166,6 +166,10 @@ The list of flags provided below may not be up-to-date due to ongoing developmen
   Comma-separated whitelist of origins for CORS, or cross-origin resource sharing, (empty or * means allow all).
 --host-whitelist '*'
   Acceptable hostnames from HTTP client requests, if server is not secure (empty or * means allow all).
+--tls-min-version 'TLS12'
```

Comment:
It's "`TLS1.2`" in https://github.com/etcd-io/etcd/pull/15156, so I suggest to keep it consistent.

Reply:
Sorry for missing that! I've pushed a fix.